src/main/java/com/puppycrawl/tools/checkstyle/DetailNodeTreeStringPrinter.java

```diff
@@ -88,7 +88,7 @@ private static DetailNode parseJavadocAsDetailNode(String javadocComment) {
      * @param parseErrorMessage ParseErrorMessage
      * @return error message
      */
-    private static String getParseErrorMessage(ParseErrorMessage parseErrorMessage) {
+    public static String getParseErrorMessage(ParseErrorMessage parseErrorMessage) {
```

Comment:
Done. 

@romani 
I couldn't find any existing public methods to generate an error message from `ParseErrorMessage` object. Another option would be to rewrite the logic in the method obtained through reflection again for the UTs. And then a `ParseErrorMessage` object won't even be necessary and we can then use `LocalizedMessage` object as is used here at [DetailNodeTreeStringPrinter.java#L92](https://github.com/checkstyle/checkstyle/blob/master/src/main/java/com/puppycrawl/tools/checkstyle/DetailNodeTreeStringPrinter.java#L92)